### PR TITLE
Create facet-filter-tags

### PR DESCRIPTION
**Pull request by heatherco**
Facet filters with filter tags

Search and state of facet filters

### CODE REVIEW

File: src/data/index/watson-health/facet-filter-tags [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url of component code on GitHub
+maintainer: name of maintainer team
+description: >-
+  short description of purpose and usage of component
+framework: choose one of Vanilla | Angular | React | Vue
+design_asset: choose one of Sketch | Axure | Adobe XD | Figma
+platform: Web
+aliases:
+  - fuzzy term for component (e.g. an alias for a "card" component might be "tile")
+date_added: YYY-MM-DDT00:00:00.000Z
```
Comment (emyarod) [inline src/data/index/watson-health/facet-filter-tags:14]: I believe these fields need to be filled out to pass CI

**Review by emyarod — COMMENTED** [review]
hi @heatherco is there a GitHub issue attached to this PR? just looking for some more context around this change

**Review by andreancardona — CHANGES_REQUESTED** [review]
@heatherco following up with @emyarod 's comment :)

**Comment by tw15egan** [comment]
Hey @heatherco, looks like there hasn't been any action taken on these PRs, so we're going to close them out for now. If you'd like these items to be added to the index, please fill out the `.yml` file and we can reopen the PR. Thanks!

**Pull request closed without merging**